src/main/markdown/doc/latest/tutorial/manageevents.md

```diff
@@ -80,40 +80,40 @@ import com.google.gwt.user.client.ui.VerticalPanel;
     stocksFlexTable.setText(0, 2, "Change");
     stocksFlexTable.setText(0, 3, "Remove");
 
-        // Assemble Add Stock panel.
+    // Assemble Add Stock panel.
     addPanel.add(newSymbolTextBox);
     addPanel.add(addStockButton);
 
-        // Assemble Main panel.
+    // Assemble Main panel.
     mainPanel.add(stocksFlexTable);
     mainPanel.add(addPanel);
     mainPanel.add(lastUpdatedLabel);
 
-        // Associate the Main panel with the HTML host page.
+    // Associate the Main panel with the HTML host page.
     RootPanel.get("stockList").add(mainPanel);
 
-        // Move cursor focus to the input box.
+    // Move cursor focus to the input box.
     newSymbolTextBox.setFocus(true);
 
-        // Listen for mouse events on the Add button.
+    // Listen for mouse events on the Add button.
     addStockButton.addClickHandler(new ClickHandler() {
       public void onClick(ClickEvent event) {
         addStock();
       }
     });
 
```

Comment:
At the same point, do you mind to remove all these empty lines at the end of blocks.